orchagent/aclorch.cpp

```diff
@@ -3304,6 +3314,21 @@ void AclOrch::doAclRuleTask(Consumer &consumer)
                     break;
                 }
             }
+            // If acl rule is to match TCP_FLAGS, and IP_PROTOCOL is not set
+            // we set IP_PROTOCOL to 6 to match TCP explicitly
+            if (bHasTCPFlag && !bHasIPProtocol)
```

Comment:
Suggest adding a VS test to cover this by only adding TCP flag and check if both fields are set in SAI

Reply:
Changes looks good to me. Second to Prince's comment.  Please add test case coverage for this change. Thanks!

Reply:
Sure, will upadte. Thanks